### PR TITLE
Scaffold an abstract generator and an EC2 data generator. Closes #4.

### LINKED ISSUE #4: Create an abstract generator and EC2 generator
Create a structure for generating the boiler plate data needed.

Create a generator for EC2 instance usage creating a random number of instance using some selected real data points.

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -12,7 +12,7 @@ omit =
   *wsgi.py
   *migrations/*
   *test/*
-  */tests_*.py
+  */test_*.py
```
Comment (blentz) [inline .coveragerc]: Would `*/test*.py` make more sense to cover all the bases? I'm not sure how consistently we want to name the tests.

File: .gitignore [hunk 2]
```diff
@@ -102,3 +102,5 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+.vscode
```
Comment (blentz) [inline .gitignore]: Needs `.sw*` for us Vim users, too.

Comment (chambridge) [reply .gitignore]: done.

File: .coveragerc (hunk 1 above)
Comment (chambridge) [reply .coveragerc]: done.

**Comment by chambridge** [comment]
@blentz pushed up changes. Can you re-approve?

File: .gitignore (hunk 2 above)
Comment (chargio) [reply .gitignore]: Shouldn't that be part of your overall .global_gitignore file?

**Pull request merged**